app/connector/FileStoreConnector.scala

```diff
@@ -79,6 +82,9 @@ class FileStoreConnector @Inject() (
 
   def initiate(request: FileStoreInitiateRequest)(implicit hc: HeaderCarrier): Future[FileStoreInitiateResponse] =
     withMetricsTimerAsync("initiate-file-upload") { _ =>
+
+      Logger.error(s"\n\n initiate-file-upload :" + request + "\n\n")
```

Comment:
`play.api.Logger` is deprecated, but you can replace it by importing `play.api.Logging` and extending `Logging` to get a `logger` member in this class